docs/user/03-11-customize_secret.md

```diff
@@ -0,0 +1,45 @@
+# Customize the Default Credentials and Access
+
+You can customize the `sap-btp-manager` Secret and manage your own SAP BTP Operator's default configuration.
+
+## Context
+
+When you create SAP BTP, Kyma runtime, the `sap-btp-manager` Secret is automatically created as the default Secret managing the SAP BTP Operator's resources. 
+Any changes to the Secret are reverted, and the previous settings are restored within up to 24 hours.
+See [Preconfigured Credentials and Access](03-10-preconfigured-secret.md#credentials).
+
+To customize the `sap-btp-manager` Secret and prevent your changes from being reverted, you must stop the Secret's reconciliation.
+With the customized Secret, you can perform the following actions:
+
+* Manage your own SAP BTP Operator's default configuration
+* Migrate the instances you created outside of the Kyma environment to your Kyma cluster
+
+## Procedure
+
+To customize the `sap-btp-manager` Secret, perform the following steps:
+
+* Label the Secret with `kyma-project.io/skip-reconciliation: 'true'`.
+* Provide the credentials of the Kyma instance you want to migrate or recreate.
+* Optionally, add the `management_namespace` parameter and provide the name of your custom management namespace.
+
+## Result
+
+Your customized `sap-btp-manager` Secret is now the default SAP BTP Operator's Secret. It generates the SAP BTP service operator's resources, as shown in the following diagram:
```

Comment:
preposition instead of genitive makes it easier to understand for me

```suggestion
Your customized `sap-btp-manager` Secret is now the default Secret of the SAP BTP Operator. It generates the SAP BTP service operator's resources, as shown in the following diagram:
```